pages/index.html

```diff
@@ -46,38 +40,36 @@ <h4 class="text-uppercase">
   <div class="grid-container">
     <div class="grid-row grid-gap-lg">
       {% for filter in site.catalog_filters %}
-        <div class="grid-col-12 tablet:grid-col-6 desktop:grid-col-3">
-          {% assign iconName = filter.iconName | escape %}
-          <a href="{% link filter.url %}">
-            <div class="icon-container text-center">
-              {% datagov_icon iconName "usa-icon--size-5"%}
-              <div class="name">{{ filter.name }}</div>
-            </div>
-          </a>
-        </div>
+      <div class="grid-col-12 tablet:grid-col-6 desktop:grid-col-3">
+        {% assign iconName = filter.iconName | escape %}
+        <a href="{% link filter.url %}">
+          <div class="icon-container text-center">
+            {% datagov_icon iconName "usa-icon--size-5"%}
+            <div class="name">{{ filter.name }}</div>
+          </div>
+        </a>
+      </div>
       {% endfor %}
     </div>
   </div>
 </section>
 
-<!-- About  -->
-<section class="usa-section about">
+<!-- Mission  -->
+<section class="usa-section mission">
   <div class="grid-container">
     <div class="grid-row grid-gap-lg">
-      <div class="grid-col-12 desktop:grid-col-4">
-        <img class="datagov-age-icon" alt="Data Gov is 14yrs old!" role="img"
-          src="https://s3-us-gov-west-1.amazonaws.com/cg-0817d6e3-93c4-4de8-8b32-da6919464e61/datagov-age-14.png" />
-      </div>
-      <div class="grid-col-12 desktop:grid-col-8">
-        <h1>About Data.gov</h1>
+      <div class="grid-col-12 desktop:grid-col-12 mission__copy">
+        <h1>Mission</h1>
         <div>
-          Data.gov is the United States government's open data website. It provides access to datasets published by agencies
-          across the federal government. Data.gov is intended to provide access to government open data to the public, achieve
-          agency missions, drive innovation, fuel economic activity, and uphold the ideals of an open and transparent government.
+          The United States Government’s open data site is designed to unleash the power of government open data to
+          <b>inform
+            decisions by the public and policymakers</b>, <b>drive innovation and economic activity</b>, <b>achieve
```

Comment:
```suggestion
            decisions of the public and policymakers</b>, <b>drive innovation and economic activity</b>, <b>achieve
```
Same as above.

Reply:
I'm gonna leave this as is. It reads fine.